ccd-definition/ComplexTypes/PartyFlagStructure-HMC-nonprod.json

```diff
@@ -5,6 +5,6 @@
     "FieldType": "Text",
     "ElementLabel": " ",
     "SecurityClassification": "Public",
-    "RetainHiddenValue": "Y"
```

Comment:
Just to note this had to be removed to allow a show condition to be added. From what i have seen if you try and put retain hidden value and show condition together, and use the complex type as part of a collection then you get a ccd error when uploading complaining about the parent casefield not containing a show condition, which does not make sense since if its not a collection we do not get this error.

So I have removed retainHidden value and done a bunch of manual tests to ensure the partyID in witnesses and experts fields persist and do not get wiped after creation later on down the line. Seems to still work as expected.